submissions/alyonafuria/html-css_popup/index.html

```diff
@@ -0,0 +1,153 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Hooli popup</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <nav>
+        <ul class="navigation">
+            <li><a href="#" class="upperButton">Mail</a></li>
+            <li><a href="#" class="upperButton">Images</a></li>
+            <li>
+
+                <input type="checkbox" id="toggle" />
```

Comment:
Your popup not accessible from keyboard. [Watch video](https://share.vidyard.com/watch/EdJpsRU4oo3Rzv8YCQEmhm?)
Please fix it.